src/main/kotlin/net/minecrell/pluginyml/bukkit/BukkitPluginDescription.kt

```diff
@@ -78,7 +78,12 @@ class BukkitPluginDescription(project: Project) : Serializable {
     data class Permission(@Transient @JsonIgnore val name: String) : Serializable {
         var description: String? = null
         var default: Default? = null
-        var children: List<String>? = null
+        var children: List<String>? // No @[Transient JsonIgnore] needed as it has no backing value
```

Comment:
FYI, this comment doesn't seem to be correct:

```
Execution failed for task ':Bukkit:generateBukkitPluginDescription'.
> com.fasterxml.jackson.databind.JsonMappingException: Conflicting getter definitions for property "children": net.minecrell.pluginyml.bukkit.BukkitPluginDescription$Permission#getChildren() vs net.minecrell.pluginyml.bukkit.BukkitPluginDescription$Permission#getChildrenMap() (through reference chain: net.minecrell.pluginyml.bukkit.BukkitPluginDescription["permissions"])
```

`@JsonIgnore` is needed on the getter to make it work.